### ISSUE TITLE
Occasional crash in simulator builds

### ISSUE THREAD

**Issue by ntorring** [MapsIndoors/MapsIndoorsIOS#4]
I'm experiencing occasional crashes when running simulator builds, about 1 in 5 launches. I have not yet experienced any crashes on device builds, which might just be coincidence!
My test application just opens up on a map view, and the crash appears to be caused by a MPVenueProvider thread.
Here is a screenshot of the stack trace for the crash (I'm getting a EXC_BAD_ACCESS crash):
![screenshot 2018-12-12 at 10 47 47](https://user-images.githubusercontent.com/7045012/49861883-34409100-fdfd-11e8-8138-b05f3299236d.png)

I'm using Xcode 10.1, with a iOS 12.1 simulator
Here's a snippet of my view controller code:
```
override func loadView() {
  super.loadView()
  MapsIndoors.provideAPIKey(AppDelegate.mApiKey, googleAPIKey: AppDelegate.gApiKey)
  let camera = GMSCameraPosition.camera(withLatitude: 57.057964, longitude: 9.9504112, zoom: 17)
  mapView = GMSMapView.map(withFrame: CGRect.zero, camera: camera)
  mapView.settings.compassButton = true
  self.view = mapView
  mapControl = MPMapControl.init(map: mapView)
}
```

**Comment by Daniel-Nielsen** [comment]
@ntorring thanks for reporting. In which SDK version are you seeing this issue?

**Comment by ntorring** [comment]
I have just tested with 2.1.7, and I'm still experiencing the same issue.

**Comment by ntorring** [comment]
It would seem that the issue is also present in release builds, it's just a lot less frequent. This is snippet of a Crashlytics stack trace:
```
Crashed: com.mapsindoors.MPVenueProvider
0    0x182da30d8 -[__NSDictionaryM setObject:forKeyedSubscript:] + 420
1    0x1009885d8 -[MPJSONModel __customSetValue:forProperty:] + 188256
2    0x100985dd0 -[MPJSONModel __importDictionary:withKeyMapper:validation:error:] + 178008
3    0x1009852b4 -[MPJSONModel initWithDictionary:error:] + 175164
4    0x1009b7d08 -[MPMultiPolygonGeometry initWithDictionary:error:] + 382608
5    0x10097243c -[MPFloor initWithDictionary:error:] + 97732
6    0x100987a00 -[MPJSONModel __transform:forProperty:error:] + 185224
7    0x100985ff8 -[MPJSONModel __importDictionary:withKeyMapper:validation:error:] + 178560
8    0x1009852b4 -[MPJSONModel initWithDictionary:error:] + 175164
9    0x1009895c0 +[MPJSONModel arrayOfModelsFromDictionaries:error:] + 192328
10   0x100987738 -[MPJSONModel __transform:forProperty:error:] + 184512
11   0x100985ff8 -[MPJSONModel __importDictionary:withKeyMapper:validation:error:] + 178560
12   0x1009852b4 -[MPJSONModel initWithDictionary:error:] + 175164
13   0x100984f8c -[MPJSONModel initWithData:error:] + 174356
14   0x100985118 -[MPJSONModel initWithString:usingEncoding:error:] + 174752
15   0x100985050 -[MPJSONModel initWithString:error:] + 174552
16   0x1009db408 -[MPVenueProvider getBuildingsAsync:arg:language:completionHandler:] + 135568
17   0x1009daddc __79-[MPVenueProvider getBuildingWithinBoundsAsync:arg:language:completionHandler:]_block_invoke + 133988
```

**Comment by Daniel-Nielsen** [comment]
Should be fixed from version 2.1.9-rc2
